react/features/base/tracks/functions.js

```diff
@@ -159,15 +159,18 @@ export function getTracksByMediaType(tracks, mediaType) {
 /**
  * Mutes or unmutes a specific <tt>JitsiLocalTrack</tt>. If the muted state of
  * the specified <tt>track</tt> is already in accord with the specified
- * <tt>muted</tt> value, then does nothing.
+ * <tt>muted</tt> value, then does nothing. If the mute/unmute operation fails,
+ * then an appropriate action to rollback the muted state will be dispatched.
  *
+ * @param {Store} store - Redux store that will be used to dispatch "rollback"
+ * action in case operation fails.
  * @param {JitsiLocalTrack} track - The <tt>JitsiLocalTrack</tt> to mute or
  * unmute.
  * @param {boolean} muted - If the specified <tt>track</tt> is to be muted, then
  * <tt>true</tt>; otherwise, <tt>false</tt>.
  * @returns {Promise}
  */
-export function setTrackMuted(track, muted) {
+export function setTrackMuted(store, track, muted) {
```

Comment:
Lenny suggested to not pass the store and not dispatch from functions, so I'm going to rollback this method.